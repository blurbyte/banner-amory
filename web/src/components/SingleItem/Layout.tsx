import styled from 'styled-components';

const Layout = styled.div`
  display: grid;
  grid-template-columns: 15rem 1fr;
  margin-top: 2rem;
`;

export default Layout;
