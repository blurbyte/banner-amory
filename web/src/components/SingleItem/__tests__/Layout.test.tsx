import * as React from 'react';
import * as renderer from 'react-test-renderer';
import 'jest-styled-components';

import Layout from '../Layout';

test('renders correctly', () => {
  const component = renderer.create(<Layout />);
  expect(component.toJSON()).toMatchSnapshot();
});
