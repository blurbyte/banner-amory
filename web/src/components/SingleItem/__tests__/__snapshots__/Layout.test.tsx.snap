// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders correctly 1`] = `
.c0 {
  display: grid;
  grid-template-columns: 15rem 1fr;
  margin-top: 1.2rem;
}

<div
  className="c0"
/>
`;
